scalableminds/webknossos
# Split Multi-Component Trees in Backend NML Parser

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- upload NML with at least one multicomponent tree (that is in the root group)
- upload NML with at least one multicomponent tree (that is in a tree group)
- test NMLs can be found at [multicomponent-test-nmls.zip](https://github.com/scalableminds/webknossos/files/4878036/multicomponent-test-nmls.zip)
- should result in two distinct trees, wrapped in a new tree group
- uploading NML with _no_ multicomponent trees should not be significantly slower
### Issues:
- fixes #4374 

------
